docs/docs/concepts/policies.mdx

```diff
@@ -378,41 +343,44 @@ The _Intentless Policy_ is part of Rasa's new Conversational AI with
-  [don't fit into intents](https://rasa.com/blog/were-a-step-closer-to-getting-rid-of-intents/)
-  and conversation context is necessary to choose a course of action.
-
-Using the `IntentlessPolicy`, a
-question-answering bot can already understanding many different ways
-that users could phrase their questions - even across a series of user messages:
-
-This only requires appropriate responses to be defined in the domain file.
-
-To eliminate hallucinations, the policy only chooses which response from
-your domain file to send. It does not generate new text.
-
-In addition, you can control the LLM by:
-
-- providing example conversations (end-to-end stories) which will be used in the prompt.
-- setting the confidence threshold to determine when the intentless policy should kick in.
-
-[This repository](https://github.com/RasaHQ/starter-pack-intentless-policy) contains a starter pack with a bot that uses the
-`IntentlessPolicy`. It's a good starting point for trying out the policy and for
-extending it.
-
-## Demo
-
-[Webinar demo](https://hubs.ly/Q01CLhyG0) showing that this policy can already
-handle some advanced linguistic phenomena out of the box.
-
-The examples in the webinar recording are also part of the end-to-end tests
-defined in the [example repository](https://github.com/RasaHQ/starter-pack-intentless-policy) in (`tests/e2e_test_stories.yml`).
-
-## Adding the Intentless Policy to your bot
+The Intentless Policy is used to send `responses` in a conversation which
+are not part of a flow. 
+This can be helpful for handling chitchat, contextual questions,
+and high-stakes topics.
+
+### Chitchat
+In an enterprise setting it may not be appropriate to use
+a purely generative model to handle chitchat.
+Teams want to ensure that the assistant is always on-brand and on-message.
+Using the Intentless Policy, you can define vetted, human-authored `responses`
+that your assistant can send.
+Because the Intentless Policy leverages LLMs and considers the whole context of the conversation
+when selecting an appropriate `response`, it is much more powerful
+than simply predicting an intent and triggering a fixed response to it. 
+
+### High-stakes Topics
+In addition to chitchat, another common use case for the Intentless Policy is
+providing answers on high-stakes topics.
+For example, if users have questions about policies, legal terms, or guarantees,
+like: _"My situation is X, I need Y. Is that covered by my policy?"_
+In these cases the DocSearchPolicy's RAG approach is risky. 
+Even with the relevant content present in the prompt, a RAG approach allows an 
+LLM to make interpretations of documents. 
+The answer users get will vary depending on the exact phrasing of their question,
+and may change when the underlying model changes. 
+
+For high-stakes topics, it is safest to send a self-contained, vetted answer
+rather than relying on a generative model.
+The Intentless Policy provides that capability in a CALM assistant.
```

Comment:
hmm, not without risk either. You might have responses
* If you have X, that is covered
but also
* yes
* no

not 100% chance that you'll get the right one. Hard to tell, but the additional context could make RAG safer in some situations.

Reply:
yes I see what you're saying. That was the point about "self-contained" answers, but agree that if `yes` and `no` are candidates for intentless then you do run this risk. So there's definitely a design aspect to it